storm-core/src/ui/public/flux.html

```diff
@@ -0,0 +1,139 @@
+<html>
```

Comment:
Missing apache license. And I guess we may need a link from storm ui homepage to flux.html.